File: src/Coding.Gtm/Gtm.Business/Domain/Managers/ProductPrice/Models/CountryProductPriceModel.cs
namespace Gtm.Business.Domain.Managers.ProductPrice.Models
{
    /// <summary>
    /// Country model
    /// </summary>
    public sealed class CountryProductPriceModel
    {
        #region Properties

        /// <summary>
        /// Country id
        /// </summary>
        public long CountryId { get; set; }
        /// <summary>
        /// Country name
        /// </summary>
        public string CountrytName { get; set; }
        /// <summary>
        /// Indicates if it is saved 
        /// </summary>
        public bool IsSaved { get; set; }
        /// <summary>
        /// Rates
        /// </summary>
        public IEnumerable<RateCountryProductPriceModel> Rates { get; set; }
        /// <summary>
        /// Price without VAT
        /// </summary>
        public decimal? PriceWithoutVAT { get; set; }
        /// <summary>
        /// Value Added Tax
        /// </summary>
        public int? VAT { get; set; }
        /// <summary>
        /// Price including VAT
        /// </summary>
        public decimal? PriceInclVAT { get; set; }

        #endregion
    }
}

File: src/Coding.Gtm/Gtm.UI/app-purchases/src/app/services/product-price/read/models/country-product-price-model.ts
import { RateCountryProductPriceModel } from "./rate-country-product-price-model";

export class CountryProductPriceModel {
	countryId: number;
    countrytName: string;
    isSaved: boolean;
    priceWithoutVAT: number;
    vat: number;
    priceInclVAT: number;
    rates: RateCountryProductPriceModel[]
}